src/components/PanelFields.tsx

```diff
@@ -242,7 +242,8 @@ export const PanelFields: React.FC<{
               mt="medium"
               onChange={(e: SyntheticEvent) => {
                 const target = e.target as HTMLInputElement
-                setSearch(target.value)}}
+                setSearch(target.value)
```

Comment:
if you use a generic, you won't have to `as` here:

```typescript
(e: SyntheticEvent<HTMLInputEvent>) => {
  setSearch(e.target.value)
}
```

Reply:
Nice call!